### DISCUSSION TITLE
AutoreloadMagics Error

### DISCUSSION THREAD

**Discussion by thatcort** [microsoft/vscode-jupyter#14622] [Questions and Answers] [answered]
I'm getting the following error every time I run a cell:

```
TypeError: AutoreloadMagics.pre_run_cell() takes 1 positional argument but 2 were given
```

Does anyone know what this means or what to do about it?

**Comment by DonJayamanne** [comment, marked as answer]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/14618

**Reply by thatcort** [reply]
Thanks
